core/src/main/java/com/microsoft/applicationinsights/internal/util/PropertyHelper.java

```diff
@@ -28,8 +30,9 @@
 /**
  * Helper class for reading data from a properties file found on the class path.
  */
-public class PropertyHelper
-{
+public final class PropertyHelper {
+    public final static String SDK_VERSION_FILE_NAME = "sdk-version.properties";
```

Comment:
I'm not sure I agree. PropertyHelper is a helper in the core context. Why have another class for that? what if we need to have another propertis file that is loaded from various places?